config/kinto.ini

```diff
@@ -10,7 +10,7 @@ cliquet.retry_after = 30
 cliquet.eos =
 
 pyramid.debug_notfound = true
-# cliquet.basic_auth_backdoor = true
+cliquet.basic_auth_backdoor = true
```

Comment:
I think we should rename it to `basic_auth_activated` since it is not really a backdoor anymore but an Auth feature.